.github/workflows/model_loading.yml

```
@@ -0,0 +1,53 @@
+name: Model Loading
+
+on:
+  pull_request:
+    paths:
+      - 'mteb/models/**.py'
+
+jobs:
+  extract-and-run:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Set up Python
+      uses: actions/setup-python@v4
+      with:
+        python-version: '3.10'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install mteb[pylate] gritlm mteb[model2vec] xformers
```

```suggestion
        pip install mteb[pylate, model2vec, gritlm, xformers]
```
assuming it should be specified as so (otherwise it could break unintentionally)

---

Also, repo doesn't have `requirements.txt` and this should be `pip install .[pylate, model2vec, gritlm, xformers]`

---

Fill to be removed.